src/js/mqtt.js

```diff
@@ -440,7 +440,7 @@ function storageTimerHit() {
     this.write(native.ping());
 
     if (this.pingrespCounter == 0) {
-      this.pingrespCounter = ((this.keepalive + 5) * 1.5) | 0;
+      this.pingrespCounter = ((this.keepalive + 5) * ('1.5' - 0)) | 0;
```

Comment:
This is a frequent code path, I would change it to:
`((this.keepalive + 5) * 3 / 2) | 0;`

Reply:
Or: `(this.keepalive + 5) * 3 >> 2;`